# These are the times for which the grid is valid
uint32 begin
uint32 end

# This is the map data
nav_msgs/OccupancyGrid map
